docs/tasks/debug-application-cluster/audit.md

```diff
@@ -22,105 +22,54 @@ answer the following questions:
-2017-03-21T03:57:09.108403639-04:00 AUDIT: id="c939d2a7-1c37-4ef1-b2f7-4ba9b1e43b53" response="200"
-```
-
-Note that Kubernetes 1.8 has switched to use the advanced structured audit log by default.
-To fallback to this legacy audit, disable the advanced auditing feature
-using the `AdvancedAuditing` feature gate on the [kube-apiserver][kube-apiserver]:
-
-```
---feature-gates=AdvancedAuditing=false
-```
-
-### Configuration
-
-[Kube-apiserver][kube-apiserver] provides the following options which are responsible
-for configuring where and how audit logs are handled:
-
-- `audit-log-path` - enables the audit log pointing to a file where the requests are being logged to, '-' means standard out.
-- `audit-log-maxage` - specifies maximum number of days to retain old audit log files based on the timestamp encoded in their filename.
-- `audit-log-maxbackup` - specifies maximum number of old audit log files to retain.
-- `audit-log-maxsize` - specifies maximum size in megabytes of the audit log file before it gets rotated. Defaults to 100MB.
-
-If an audit log file already exists, Kubernetes appends new audit logs to that file.
-Otherwise, Kubernetes creates an audit log file at the location you specified in
-`audit-log-path`. If the audit log file exceeds the size you specify in `audit-log-maxsize`,
-Kubernetes will rename the current log file by appending the current timestamp on
-the file name (before the file extension) and create a new audit log file.
-Kubernetes may delete old log files when creating a new log file; you can configure
-how many files are retained and how old they can be by specifying the `audit-log-maxbackup`
-and `audit-log-maxage` options.
-
-## Advanced audit
-
-Kubernetes 1.7 expands auditing with experimental functionality such as event
-filtering and a webhook for integration with external systems. Kubernetes 1.8
-upgrades the advanced audit feature to beta, and some backward incompatible changes
-have been committed.
+[Kube-apiserver][kube-apiserver] performs auditing. Each request on each stage
+of its execution generates an event, which is then pre-processed according to
+a certain policy and written to a backend. You can find more details about the
+pipeline in the [design proposal][auditing-proposal].
 
+## Audit Policy
 
-`AdvancedAuditing` is customizable in two ways. Policy, which determines what's recorded,
-and backends, which persist records. Backend implementations include logs files and
-webhooks.
+Audit policy defines rules about what events should be recorded and what data
+they should include. When an event is processed, it's compared against the list
+of rules in order. The first matching rule sets the [audit level][auditing-level]
+of the event. The audit policy object structure is defined in the
+[`audit.k8s.io` API group][audit-api].
```

Comment:
This will not be transformed I think.
Should it be ```[`audit.k8s.io` API group][auditing-api]```?

Reply:
A link would be very helpful here.

Reply:
Thanks! Done